css/index.css

```diff
@@ -1,9 +1,223 @@
-/* Example of @import use
-This imports the `reset.css` file */
-@import "./reset.css";
-
-/* For testing purposes, delete this code */
-body {
-  font-size: max(20vw, 400px);
-  width: min(20vw, 400px);
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.primary-bg {
+  color: white;
+  background-color: yellow;
+}
+
+.edit-me {
+  background-color: #1f2937;
+  height: 500px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.header-container {
+  width: 80px;
+  height: 80px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.header-nav-container {
+  display: flex;
+  justify-content: space-between;
+  width: 50%;
+}
+
+.header-nav-container li {
+  list-style-type: none;
+}
+
+.header-logo {
+  width: 40%;
+}
+
+.flex {
+  width: 40%;
+}
+
+.main-hero {
+  display: flex;
+}
+
+.second-header-container {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 80%;
+  height: 300px;
+}
+
+.section-header {
+  font-family: "Franklin Gothic Medium", "Arial Narrow", Arial, sans-serif;
+  font-size: 40px;
+}
+
+.section-info {
+  font-family: "Franklin Gothic Medium", "Arial Narrow", Arial, sans-serif;
+  font-size: 14px;
+}
+
+.web-info {
+  width: 30%;
+}
+
+.btn {
+  border-style: none;
+  background-color: blue;
+  color: white;
+  border-radius: 10px;
+  width: 100px;
+  height: 30px;
+  margin-top: 10px;
+}
+
+.image-section {
+  width: 40%;
+  height: 80%;
+  background-color: grey;
+  border: 50px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.section-a {
+  background-color: white;
+  width: 100%;
+  height: 350px;
+}
+
+.section-b {
+  height: 80%;
+  width: 600px;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  margin: auto;
+}
+
+.section-header-b {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  font-family: "Franklin Gothic Medium", "Arial Narrow", Arial, sans-serif;
+  font-size: 30px;
+  margin-top: 35px;
+}
+
+.alt {
+  border: 3px solid blue;
+  width: 85%;
+  height: 130px;
+  margin-top: 20px;
+  border-radius: 10px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+p {
```

Comment:
Please let do away from styling element, let's give them class and style the class